### ISSUE TITLE
embeded images takes lots of memory

### ISSUE THREAD

**Issue by kw-user123** [kiwiirc/kiwiirc#1308]
For unknown reason just 80 embedded pictures took over 1.5GB of firefox browser process memory. Pictures were 400kb on average.

It is not an browser issue because I have seen other chat systems where they are also embeding images and they had no issue with over 500 pics in one window.

**Comment by kw-user123** [comment]
I thought that this issue was related to browser console error with platform.js script or with system memory limits. Today I have tried to decreased scrollback_size from 250 even to 20. That caused this issue to appeared more quickly (tried several times).

Then I have increased scrollback_size to 2000 lines and now couple of hours later there are no issues with browser memory sizes or with disappearance of images which both occured quickly if scrollback_size is set to low value.

Tried this in Chrome and Firefox in Windows, with the same results (using current kiwi nextclient).
